### ISSUE TITLE
Re-Ranking On FB15K237

### ISSUE THREAD

**Issue by yw3l** [intfloat/SimKGC#41]
Hi, @intfloat 
        I would like to know if your results on the FB15K237 dataset exclude triples that have appeared in the training set in the test set ?
        I would appreciate your help! Looking forward to your reply！
